install_config/router/f5_router.adoc

```diff
@@ -272,17 +270,17 @@ node_].
 ====
 
 ifdef::openshift-enterprise[]
```

Comment:
@vikram-redhat, are we far enough from version 3.4 to remove this ifdef and just use the origin text?

Reply:
@pdp2shirts @kalexand-rh - yes, makes sense.

Reply:
Then please remove lines 272-277 and 281.